src/components/card/card.less

```diff
@@ -11,54 +11,62 @@
   box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.3);
 
   .avatar{
-    margin-right: 15px;
+    margin-right: 10px;
     img{
       display: block;
     }
   }
 
   .thumbnail {
-    padding: 6px 0 0 0;
+    padding: 0;
     height: 0;
-    padding-bottom: 61.875%;
+    padding-bottom: 64.875%;
```

Comment:
Yes, the 3% increase was intended for the reason you say above. I can check just a normal px or rem value and see what/if it breaks.